python/ml4ir/base/model/architectures/fixed_additive_positional_bias.py

```diff
@@ -0,0 +1,60 @@
+import tensorflow as tf
```

Comment:
@mohazahran You should move this to under ranking directory.

Reply:
Do you have specific dir in mind because to me it fits under architecture.